homework_1/task2/TaxLibrary/tax.go

```diff
@@ -0,0 +1,88 @@
+package TaxLibrary
+
+import (
+	"math"
+
+	"github.com/pkg/errors"
+)
+
+type TaxBracketsConfiguration struct {
+	Brackets   []float64
+	Percentage []float64
+}
```

Comment:
By having `brackets` and `percentage` separated you have to assert whether the lengths of those two arrays are equal etc... Why not simply enforce that via types immediately? Something like:

```suggestion
type TaxBracket struct {
  Threshold float64
  Percentage float64
}
```

And then your functions can receive `[]TaxBracket` as configuration and you don't need to check whether every bracket has a percentage, it's already enforce by the type-system.